### PR TITLE
libbcc-py: Fix libbpf types

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
1ad2656a1d9c ("Add support_kfunc function to BPF object") added new functions
to libbcc-py but didn't set the restype and argstypes for those. It's causing
the `bpf_has_kernel_btf` function to return True in systems without support
for BTF, making tools like opensnoop, klockstat and any other using kfuncs
unusable in those systems.

The following Python script reproduces the problem:

```
from bcc import BPF
print(BPF.support_kfunc())
```

Signed-off-by: Mauricio Vásquez <mauricio@kinvolk.io>

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: src/python/bcc/libbcc.py [hunk 1]
```diff
@@ -108,6 +108,12 @@
 lib.bpf_detach_tracepoint.argtypes = [ct.c_char_p, ct.c_char_p]
 lib.bpf_attach_raw_tracepoint.restype = ct.c_int
 lib.bpf_attach_raw_tracepoint.argtypes = [ct.c_int, ct.c_char_p]
+lib.bpf_detach_kfunc.restype = ct.c_int
+lib.bpf_detach_kfunc.argtypes = [ct.c_int, ct.c_char_p]
```
Comment (yonghong-song) [inline src/python/bcc/libbcc.py:112]: Looks good. Could you remove `bpf_detach_kfunc`? In the python side, it is not needed as python just close the fd, no need to go to C++.

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: src/python/bcc/libbcc.py (hunk 1 above)
Comment (mauriciovasquezbernal) [reply src/python/bcc/libbcc.py]: Done!

**Pull request merged**